INFO:torch.distributed.nn.jit.instantiator:Created a temporary directory at /tmp/tmpy20saeqa
INFO:torch.distributed.nn.jit.instantiator:Writing /tmp/tmpy20saeqa/_remote_module_non_scriptable.py
DEBUG:git.cmd:Popen(['git', 'version'], cwd=/home/yanmy/enrich_release, universal_newlines=False, shell=None, istream=None)
DEBUG:git.cmd:Popen(['git', 'version'], cwd=/home/yanmy/enrich_release, universal_newlines=False, shell=None, istream=None)
DEBUG:wandb.docker.auth:Trying paths: ['/home/yanmy/.docker/config.json', '/home/yanmy/.dockercfg']
DEBUG:wandb.docker.auth:No config file found
2023-12-24 18:35:18,353 - INFO - episode_0 Task: wdc-K=9 f1-score: 0.5964172813487882, path: [0, 1, 2, 3, 4, 5, 6, 7, 8, 265, 266, 355, 356, 369, 370, 205, 206, 245, 246]
INFO:my_logger:episode_0 Task: wdc-K=9 f1-score: 0.5964172813487882, path: [0, 1, 2, 3, 4, 5, 6, 7, 8, 265, 266, 355, 356, 369, 370, 205, 206, 245, 246]
DEBUG:filelock:Attempting to acquire lock 139861421828704 on /home/yanmy/.triton/autotune/Fp16Matmul_2d_kernel.pickle.lock
DEBUG:filelock:Lock 139861421828704 acquired on /home/yanmy/.triton/autotune/Fp16Matmul_2d_kernel.pickle.lock
DEBUG:filelock:Attempting to release lock 139861421828704 on /home/yanmy/.triton/autotune/Fp16Matmul_2d_kernel.pickle.lock
DEBUG:filelock:Lock 139861421828704 released on /home/yanmy/.triton/autotune/Fp16Matmul_2d_kernel.pickle.lock
DEBUG:filelock:Attempting to acquire lock 139861421828704 on /home/yanmy/.triton/autotune/Fp16Matmul_4d_kernel.pickle.lock
DEBUG:filelock:Lock 139861421828704 acquired on /home/yanmy/.triton/autotune/Fp16Matmul_4d_kernel.pickle.lock
DEBUG:filelock:Attempting to release lock 139861421828704 on /home/yanmy/.triton/autotune/Fp16Matmul_4d_kernel.pickle.lock
DEBUG:filelock:Lock 139861421828704 released on /home/yanmy/.triton/autotune/Fp16Matmul_4d_kernel.pickle.lock
